docs/setup/settings.asciidoc

```diff
@@ -429,6 +429,12 @@ to display map tiles in tilemap visualizations. By default,
 override this parameter to use their own Tile Map Service. For example:
 `"https://tiles.elastic.co/v2/default/{z}/{x}/{y}.png?elastic_tile_service_tos=agree&my_app_name=kibana"`
 
+| `migrations.batchSize:`
+ | Defines the number of documents migrated at a time. The higher the value, the faster the Saved Objects migration process will be performed at the cost of higher memory consumption. Should migration fail with the `circuit_breaking_exception` exception, you need to set a smaller `batchSize` value. *Default: `1000`*
+
+| `migrations.enableV2:`
+ | experimental[]. Enables the new Saved Objects migration algorithm. See <<upgrade-migrations>> for details. The setting is going to removed from `v8.0` when `migrations v2` becomes stable. *Default: `true`*
```

Comment:
nit: Could consider explicitly saying something along the lines of "disabling this setting will cause Kibana to use the legacy migration algorithm which shipped in 7.11 and earlier", since reverting to legacy is the only reason someone would need to touch this.